packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1120,3 +1132,26 @@ export async function pushDataToQueue(data: UploadType, requestQueueHandler: Req
 }
 
 export const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+
+export async function uploadLogs(user: string | undefined, key: string | undefined, clientBuildUuid: string) {
+    const fileStream = fs.createReadStream(BStackLogger.logFilePath)
```

Comment:
It's better to take the logs file path from argument, as this is a utility function it should depend on only its arguments. Also I feel this method is better is suited in `bstackLogger.ts`. Any reason to keep it here?